app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
@jcoyne Are you thinking this job would do the various `create` steps as `.perform_now`s?  Right now we have entirely parallelized asynchronous jobs.  I agree somewhere we would have to sacrifice that parallelization to get logic about prior tasks completion.  Certainly tacking it on here w/o that assurance is incorrect.  

Alternatively, creation of default admin set could be tied to the *next* action of repo creation, namely creation of the first user account.  The logical cost is the user model suddenly having to know a lot about fcrepo, admin sets, and solr.  That seems unwieldy.

Allow me to suggest that we may need an additional stop at a "system status" screen with long-polling for the results of the other async jobs.  Add a status line for "default admin set" (status: needed).  Then when all the rest are green, enable a button to create one.  It can say "Next" and bounce the user to the user creation, for all I care, the point is to give the user feedback and give the task the conditions it needs to succeed.